### ISSUE TITLE
Add the toggle hover mode in plotly

### ISSUE THREAD

**Issue by synackSA** [freqtrade/freqtrade#5209]
Seems the removed the toggle hover mode buttons on the plotly graph by default in recent versions, it can be added back with the following seeing added:
```
fig.update_layout(modebar_add=["v1hovermode", "toggleSpikeLines"]))
```

Is this something we could add in, I found it personally very useful.

**Comment by xmatthias** [comment]
To be honest, i'm not quite clear what you mean with "toggle hover mode".

if it's something that made sense for sure we can add it (if it's now become optional) - but first - i need to understand what it actually is :laughing:

**Comment by synackSA** [comment]
Yeah, that's my bad, I don't know what I was thinking. I actually downloaded the source with the intention of opening a PR myself, but for some reason, when I install and run everything from the source and not via docker, this isn't an issue. 

To be clear, this is what I'm talking about:
![HoverIcons](https://user-images.githubusercontent.com/5345876/123803802-988fa700-d8ba-11eb-92be-7fc09001aa45.png)
The circled icons were missing after I did a `docker pull` and then `docker-compose build`. However, as I said, I'm not sure why it's not there when I run everything via docker. I'm going to look into it some more when I get some time.
